tests/GoDaddyClientTest.php

```diff
@@ -99,7 +102,62 @@ public function testReplaceDNSRecordsForTypeHappyPath()
         self::assertEquals(
             new Google\Protobuf\GPBEmpty(),
             $resp,
-            'expected response to be GPBEmpty()',
+            'expected response to be GPBEmpty()'
+        );
+    }
+
+    public function testPatchDomainHappyPath()
+    {
+        $environment = getenv("ENVIRONMENT");
+        if ($environment == null) {
+            $environment = "DEMO";
+        }
+        $client = new GodaddyClient($environment);
+
+        $req = new PatchDomainRequest();
+        $req->setDomain("example.com");
+        $fieldMask = new FieldMask();
+        $paths = ["locked"];
+        $fieldMask->setPaths($paths);
+        $req->setFieldMask($fieldMask);
+        $req->setLocked(false);
+
+        try {
+            $resp = $client->PatchDomain($req);
+        } catch (Vendasta\Vax\SDKException $e) {
+            self::fail($e);
+            return;
+        }
+
+        self::assertEquals(
+            new Google\Protobuf\GPBEmpty(),
+            $resp,
+            'expected response to be GPBEmpty()'
+        );
+    }
+
+    public function testGetDomain()
+    {
+        $environment = getenv("ENVIRONMENT");
+        if ($environment == null) {
+            $environment = "DEMO";
+        }
+        $client = new GodaddyClient($environment);
+
+        $req = new GetDomainRequest();
+        $req->setDomain("example.com");
+
+        try {
+            $resp = $client->GetDomain($req);
+        } catch (Vendasta\Vax\SDKException $e) {
+            self::fail($e);
+            return;
+        }
+
+        self::assertEquals(
+            new Google\Protobuf\GPBEmpty(),
+            $resp,
+            'expected response to be GPBEmpty()'
```

Comment:
Should this test be expecting to get a domain rather than `GPBEmpty()`?

Reply:
Yeah, it should. I updated it.